cli/internal/helm/loader_test.go

```diff
@@ -50,6 +50,17 @@ func TestLoad(t *testing.T) {
 	assert.NotNil(chart.Dependencies())
 }
 
+func BenchmarkLoad(b *testing.B) {
+	require := require.New(b)
```

Comment:
Not sure how helpful this test is as the helm loading is not a performance critical step, but we can keep it if you want